### ISSUE TITLE
Gather feedback on printing proofs to stdout

### ISSUE THREAD

**Issue by phated** [noir-lang/noir#2109]
In #1992, we removed the ability for `nargo prove` to print a proof on stdout and instead it is always saved to the filesystem.

We are looking for feedback on this change. Do you have any uses cases that would need the proof printed on stdout?

**Comment by Savio-Sou** [comment]
cc @critesjosh @signorecello

**Comment by critesjosh** [comment]
I haven't encountered a need for it to be printed.

**Comment by critesjosh** [comment]
I've heard reports of devs finding it confusing that `nargo prove x` does not print anything when it runs successfully. Maybe printing something like "Proof ran successfully in X ms. Stored at target/x.json." on a successful run would help reduce confusion.

**Comment by signorecello** [comment]
I agree with @critesjosh we should give some feedback when proving, currently it just hangs and it can be confusing. For example, if it needs to download the backend and my connection is slow, I'm left in the dark on what exactly is happening.

@Savio-Sou can we reopen?

**Comment by Savio-Sou** [comment]
Those sound like two new Issues versus this (printing the proof upon proving).

Instead of reopening, @signorecello could you help create two new ones for:
1. Add successful message for `nargo prove` and `nargo verify`
2. Add downloading message in Nargo for backend and SRS downloads
